conventions/src/main/kotlin/otel.java-conventions.gradle.kts

```diff
@@ -186,6 +186,9 @@ testing {
   }
 }
 
+// module name
+val moduleName: String? by extra
```

Comment:
Instead of using `extra`, can you add that to the `OtelJavaExtension`? Also, consider naming it `javaModuleName` or `javaModule` to signify that it's the Java 9 module, not some arbitrary module name (e.g. gradle module).